dask/array/tests/test_array_core.py

```diff
@@ -617,6 +617,22 @@ def test_store():
     assert raises(ValueError, lambda: store([at, bt], [at, bt]))
 
 
+def test_to_hdf5():
+    try:
+        import h5py
+    except ImportError:
+        return
```

Comment:
It's a test only dependency available on conda and pypi. Up to you, but really not too bad in my opinion. It was a little bit of work up front, but really hasn't been much trouble at all for xray.

I think there is significant value to showing skipped tests -- it makes it very obvious to users running the test suite if they are getting complete coverage or not (e.g., because they have some missing dependencies).

If somebody is still running Python 2.6, they are used to annoyances like needing to install unittest2 before running test suites.